interface/src/app/mlmodel/selector.tpl.html

```diff
@@ -0,0 +1,13 @@
+<div class="well">
+  <h3>
+    Machine Learning Model
+  </h3>
```

Comment:
My preference would be to keep the actual component compact and easily "insertable," while leaving things like headers and `class="well"` up to the page that does the integrating. I haven't been great about doing this in my own pages yet, but I'd like us to start moving towards a cleaner separation of concerns between *modules* as building blocks and *layouts*, where those building blocks are assembled into a unified view or whole page.